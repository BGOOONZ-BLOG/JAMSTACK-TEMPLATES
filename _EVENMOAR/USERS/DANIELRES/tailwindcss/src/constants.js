import path from 'path'

export const cli = 'tailwind'
export const defaultConfigFile = './tailwind.config.js'
export const defaultConfigStubFile = path.resolve(__dirname, '../stubs/defaultConfig.stub.js')
export const simpleConfigStubFile = path.resolve(__dirname, '../stubs/simpleConfig.stub.js')
