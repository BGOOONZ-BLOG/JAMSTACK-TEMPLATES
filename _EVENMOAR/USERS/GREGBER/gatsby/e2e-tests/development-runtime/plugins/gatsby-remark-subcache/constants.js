exports.id = `gatsby-remark-subcache-value`
