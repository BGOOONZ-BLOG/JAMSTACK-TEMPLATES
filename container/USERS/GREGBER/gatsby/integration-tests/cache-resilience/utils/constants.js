module.exports = {
  ON_PRE_BOOTSTRAP_FILE_PATH: `./on_pre_bootstrap.state`,
  ON_POST_BUILD_FILE_PATH: `./on_post_build.state`,
}
