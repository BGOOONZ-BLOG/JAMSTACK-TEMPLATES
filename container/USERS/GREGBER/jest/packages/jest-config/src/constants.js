/**
 * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @flow
 */

import path from 'path';

export const NODE_MODULES = path.sep + 'node_modules' + path.sep;
export const DEFAULT_JS_PATTERN = '^.+\\.jsx?$';
export const DEFAULT_REPORTER_LABEL = 'default';
export const PACKAGE_JSON = 'package.json';
export const JEST_CONFIG = 'jest.config.js';
