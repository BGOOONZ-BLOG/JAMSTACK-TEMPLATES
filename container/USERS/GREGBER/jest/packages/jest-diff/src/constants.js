/**
 * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @flow
 */

import chalk from 'chalk';

export const NO_DIFF_MESSAGE = chalk.dim(
  'Compared values have no visual difference.',
);

export const SIMILAR_MESSAGE = chalk.dim(
  'Compared values serialize to the same structure.\n' +
    'Printing internal object structure without calling `toJSON` instead.',
);
