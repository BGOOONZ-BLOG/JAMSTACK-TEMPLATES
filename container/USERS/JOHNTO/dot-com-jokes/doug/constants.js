export const dougisms = [
  'No, of course not.',
  'Fuck no.',
  'Ummm... No.',
  'I liked it better before.',
  "That's bullshit.",
  "I don't give a shit.",
  'I am losing the ability to tell when you guys are joking on this stuff.',
  'Fuck it.',
  'Sigh.',
  "It's for hipsters.",
  "No, you're wrong.",
  'That looks awful.',
  "Yeah, if you're a moron.",
  'Oh, I thought I was speaking to adults...',
  "If it's new, you like it. God damn hipster.",
  "It's ruined.",
  "That's unusable.",
  'Only a child would like that.',
  "That really sounds like something I don't give a shit about.",
  'No.',
  "It's good if you like things that suck.",
  "I'm having an internal debate with myself as to whether to accept this constraint as a given or melt down in a fire of ridiculousness.",
  'Nine levels of my brain just seg-faulted at once.',
  'Fuck me with a chainsaw.',
  'I just want something usable.',
  "I don't take musical advice from someone whose musical taste sounds like two toddlers whining for more pudding.",
  "Sometimes things suck so much that they're not work appropriate.",
  'Fuck a duck.'
]

export const css = `
body { margin: 0; }
.Typist .Cursor { display: inline-block; }
.Typist .Cursor--blinking {
  opacity: 1;
  animation: blink 1s linear infinite;
}
@keyframes blink {
  0% { opacity: 1; }
  50% { opacity: 0; }
  100% { opacity: 1; }
}
`
