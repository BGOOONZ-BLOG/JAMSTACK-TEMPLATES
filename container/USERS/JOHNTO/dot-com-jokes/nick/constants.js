export const stuff = [
  'Blue Bottle pour over single cupper, made in Japan',
  'Coffee: 25g dose. 5 rounds of 70g of water, with 30 seconds in between.',
  'Beep boop music',
  'Sous vide',
  'Keyclacks',
  'Leopold mechanical keyboard with Topre switches, custom spacebar, and a LandCaps Esc key',
  'Vim with Vundle, vim-fugitive, Ctrl+P, vim-rails, vim-airline, TagHighlight, vim-flavored-markdown, and vim-markdown-preview',
  'Uniqlo UT Graphic Tees featuring Blizzard\'s award-winning game franchises',
  'spaces',
  'croquetas',
  'DoorDash',
  'ENO OneLink Shelter System with DoubleNest Hammock',
  'Klymit Static V Lightweight Sleeping Pad',
  'GoLite UltraLite +20 Quilt Sleeping Bag'
]

export const css = `
body { margin: 0; }
.Typist .Cursor { display: inline-block; }
.Typist .Cursor--blinking {
  opacity: 1;
  animation: blink 1s linear infinite;
}
@keyframes blink {
  0% { opacity: 1; }
  50% { opacity: 0; }
  100% { opacity: 1; }
}
`
