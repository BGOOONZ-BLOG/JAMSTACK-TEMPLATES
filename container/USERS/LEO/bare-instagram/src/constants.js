export const USE_NAMES_DEFAULT = [
  {
    username: 'lekoarts.de',
    picture:
      'https://scontent-muc2-1.cdninstagram.com/v/t51.2885-19/s150x150/12965119_530001620503034_1886533137_a.jpg?_nc_ht=scontent-muc2-1.cdninstagram.com&_nc_ohc=96YIguP0MwsAX8DictO&tp=1&oh=2d7186cfaad907d3bbfd91c4bb6b212d&oe=6013CCC2',
    id: '2315642426',
  },
]

export const USE_DESIGN_DEFAULT = {
  minimal: false,
  gapless: false,
  debug: false,
}
