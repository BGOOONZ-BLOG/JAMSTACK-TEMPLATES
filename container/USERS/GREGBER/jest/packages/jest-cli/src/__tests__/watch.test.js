/**
 * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 */

'use strict';

import chalk from 'chalk';
import TestWatcher from '../test_watcher';
import JestHooks from '../jest_hooks';
import {KEYS} from '../constants';

const runJestMock = jest.fn();
const watchPluginPath = `${__dirname}/__fixtures__/watch_plugin`;
const watchPlugin2Path = `${__dirname}/__fixtures__/watch_plugin2`;
let results;

jest.mock(
  '../search_source',
  () =>
    class {
      constructor(context) {
        this._context = context;
      }

      findMatchingTests(pattern) {
        const paths = [
          './path/to/file1-test.js',
          './path/to/file2-test.js',
        ].filter(path => path.match(pattern));

        return {
          tests: paths.map(path => ({
            context: this._context,
            duration: null,
            path,
          })),
        };
      }
    },
);

jest.doMock('chalk', () => new chalk.constructor({enabled: false}));
jest.doMock(
  '../run_jest',
  () =>
    function() {
      const args = Array.from(arguments);
      const [{onComplete}] = args;
      runJestMock.apply(null, args);

      // Call the callback
      onComplete(results);

      return Promise.resolve();
    },
);

jest.doMock(
  watchPluginPath,
  () =>
    class WatchPlugin1 {
      getUsageInfo() {
        return {
          key: 's'.codePointAt(0),
          prompt: 'do nothing',
        };
      }
    },
  {virtual: true},
);

jest.doMock(
  watchPlugin2Path,
  () =>
    class WatchPlugin2 {
      getUsageInfo() {
        return {
          key: 'u'.codePointAt(0),
          prompt: 'do something else',
        };
      }
    },
  {virtual: true},
);

const watch = require('../watch').default;

const nextTick = () => new Promise(res => process.nextTick(res));
const toHex = char => Number(char.charCodeAt(0)).toString(16);

afterEach(runJestMock.mockReset);

describe('Watch mode flows', () => {
  let pipe;
  let hasteMapInstances;
  let globalConfig;
  let contexts;
  let stdin;

  beforeEach(() => {
    const config = {roots: [], testPathIgnorePatterns: [], testRegex: ''};
    pipe = {write: jest.fn()};
    globalConfig = {watch: true};
    hasteMapInstances = [{on: () => {}}];
    contexts = [{config}];
    stdin = new MockStdin();
    results = {snapshot: {}};
  });

  it('Correctly passing test path pattern', () => {
    globalConfig.testPathPattern = 'test-*';

    watch(globalConfig, contexts, pipe, hasteMapInstances, stdin);

    expect(runJestMock.mock.calls[0][0]).toMatchObject({
      contexts,
      globalConfig,
      onComplete: expect.any(Function),
      outputStream: pipe,
      testWatcher: new TestWatcher({isWatchMode: true}),
    });
  });

  it('Correctly passing test name pattern', () => {
    globalConfig.testNamePattern = 'test-*';

    watch(globalConfig, contexts, pipe, hasteMapInstances, stdin);

    expect(runJestMock.mock.calls[0][0]).toMatchObject({
      contexts,
      globalConfig,
      onComplete: expect.any(Function),
      outputStream: pipe,
      testWatcher: new TestWatcher({isWatchMode: true}),
    });
  });

  it('Runs Jest once by default and shows usage', () => {
    jest.unmock('jest-util');
    const util = require('jest-util');
    util.isInteractive = true;

    const ci_watch = require('../watch').default;
    ci_watch(globalConfig, contexts, pipe, hasteMapInstances, stdin);
    expect(runJestMock.mock.calls[0][0]).toMatchObject({
      contexts,
      globalConfig,
      onComplete: expect.any(Function),
      outputStream: pipe,
      testWatcher: new TestWatcher({isWatchMode: true}),
    });
    expect(pipe.write.mock.calls.reverse()[0]).toMatchSnapshot();
  });

  it('Runs Jest in a non-interactive environment not showing usage', () => {
    jest.unmock('jest-util');
    const util = require('jest-util');
    util.isInteractive = false;

    const ci_watch = require('../watch').default;
    ci_watch(globalConfig, contexts, pipe, hasteMapInstances, stdin);
    expect(runJestMock.mock.calls[0][0]).toMatchObject({
      contexts,
      globalConfig,
      onComplete: expect.any(Function),
      outputStream: pipe,
      testWatcher: new TestWatcher({isWatchMode: true}),
    });
    expect(pipe.write.mock.calls.reverse()[0]).toMatchSnapshot();
  });

  it('resolves relative to the package root', () => {
    expect(async () => {
      await watch(
        Object.assign({}, globalConfig, {
          rootDir: __dirname,
          watchPlugins: [watchPluginPath],
        }),
        contexts,
        pipe,
        hasteMapInstances,
        stdin,
      );
    }).not.toThrow();
  });

  it('shows prompts for WatchPlugins in alphabetical order', async () => {
    jest.unmock('jest-util');
    const util = require('jest-util');
    util.isInteractive = true;

    const ci_watch = require('../watch').default;
    ci_watch(
      Object.assign({}, globalConfig, {
        rootDir: __dirname,
        watchPlugins: [watchPlugin2Path, watchPluginPath],
      }),
      contexts,
      pipe,
      hasteMapInstances,
      stdin,
    );

    const pipeMockCalls = pipe.write.mock.calls;

    const determiningTestsToRun = pipeMockCalls.findIndex(
      ([c]) => c === 'Determining test suites to run...',
    );

    expect(pipeMockCalls.slice(determiningTestsToRun + 1)).toMatchSnapshot();
  });

  it('shows update snapshot prompt (without interactive)', async () => {
    jest.unmock('jest-util');
    const util = require('jest-util');
    util.isInteractive = true;
    results = {snapshot: {failure: true}};

    const ci_watch = require('../watch').default;
    ci_watch(
      Object.assign({}, globalConfig, {
        rootDir: __dirname,
        watchPlugins: [],
      }),
      contexts,
      pipe,
      hasteMapInstances,
      stdin,
    );

    const pipeMockCalls = pipe.write.mock.calls;

    const determiningTestsToRun = pipeMockCalls.findIndex(
      ([c]) => c === 'Determining test suites to run...',
    );

    expect(pipeMockCalls.slice(determiningTestsToRun + 1)).toMatchSnapshot();
  });

  it('shows update snapshot prompt (with interactive)', async () => {
    jest.unmock('jest-util');
    const util = require('jest-util');
    util.isInteractive = true;
    util.getFailedSnapshotTests = jest.fn(() => ['test.js']);
    results = {snapshot: {failure: true}};

    const ci_watch = require('../watch').default;
    ci_watch(
      Object.assign({}, globalConfig, {
        rootDir: __dirname,
        watchPlugins: [],
      }),
      contexts,
      pipe,
      hasteMapInstances,
      stdin,
    );

    const pipeMockCalls = pipe.write.mock.calls;

    const determiningTestsToRun = pipeMockCalls.findIndex(
      ([c]) => c === 'Determining test suites to run...',
    );

    expect(pipeMockCalls.slice(determiningTestsToRun + 1)).toMatchSnapshot();
  });

  it('allows WatchPlugins to hook into JestHooks', async () => {
    const apply = jest.fn();
    const pluginPath = `${__dirname}/__fixtures__/plugin_path_register`;
    jest.doMock(
      pluginPath,
      () =>
        class WatchPlugin {
          constructor() {
            this.apply = apply;
          }
        },
      {virtual: true},
    );

    watch(
      Object.assign({}, globalConfig, {
        rootDir: __dirname,
        watchPlugins: [pluginPath],
      }),
      contexts,
      pipe,
      hasteMapInstances,
      stdin,
    );

    await nextTick();

    expect(apply).toHaveBeenCalled();
  });

  it('allows WatchPlugins to hook into file system changes', async () => {
    const fileChange = jest.fn();
    const pluginPath = `${__dirname}/__fixtures__/plugin_path_fs_change`;
    jest.doMock(
      pluginPath,
      () =>
        class WatchPlugin {
          apply(jestHooks) {
            jestHooks.fileChange(fileChange);
          }
        },
      {virtual: true},
    );

    watch(
      Object.assign({}, globalConfig, {
        rootDir: __dirname,
        watchPlugins: [pluginPath],
      }),
      contexts,
      pipe,
      hasteMapInstances,
      stdin,
    );

    expect(fileChange).toHaveBeenCalledWith({
      projects: [
        {
          config: contexts[0].config,
          testPaths: ['./path/to/file1-test.js', './path/to/file2-test.js'],
        },
      ],
    });
  });

  it('triggers enter on a WatchPlugin when its key is pressed', async () => {
    const run = jest.fn(() => Promise.resolve());
    const pluginPath = `${__dirname}/__fixtures__/plugin_path`;
    jest.doMock(
      pluginPath,
      () =>
        class WatchPlugin1 {
          constructor() {
            this.run = run;
          }
          getUsageInfo() {
            return {
              key: 's'.codePointAt(0),
              prompt: 'do nothing',
            };
          }
        },
      {virtual: true},
    );

    watch(
      Object.assign({}, globalConfig, {
        rootDir: __dirname,
        watchPlugins: [pluginPath],
      }),
      contexts,
      pipe,
      hasteMapInstances,
      stdin,
    );

    stdin.emit(Number('s'.charCodeAt(0)).toString(16));

    await nextTick();

    expect(run).toHaveBeenCalled();
  });

  it('prevents Jest from handling keys when active and returns control when end is called', async () => {
    let resolveShowPrompt;
    const run = jest.fn(() => new Promise(res => (resolveShowPrompt = res)));
    const pluginPath = `${__dirname}/__fixtures__/plugin_path_1`;
    jest.doMock(
      pluginPath,
      () =>
        class WatchPlugin1 {
          constructor() {
            this.run = run;
          }
          onKey() {}
          getUsageInfo() {
            return {
              key: 's'.codePointAt(0),
              prompt: 'do nothing',
            };
          }
        },
      {virtual: true},
    );

    const showPrompt2 = jest.fn(() => Promise.resolve());
    const pluginPath2 = `${__dirname}/__fixtures__/plugin_path_2`;
    jest.doMock(
      pluginPath2,
      () =>
        class WatchPlugin1 {
          constructor() {
            this.run = showPrompt2;
          }
          onKey() {}
          getUsageInfo() {
            return {
              key: 'z'.codePointAt(0),
              prompt: 'also do nothing',
            };
          }
        },
      {virtual: true},
    );

    watch(
      Object.assign({}, globalConfig, {
        rootDir: __dirname,
        watchPlugins: [pluginPath, pluginPath2],
      }),
      contexts,
      pipe,
      hasteMapInstances,
      stdin,
    );

    stdin.emit(Number('s'.charCodeAt(0)).toString(16));
    await nextTick();
    expect(run).toHaveBeenCalled();
    stdin.emit(Number('z'.charCodeAt(0)).toString(16));
    await nextTick();
    expect(showPrompt2).not.toHaveBeenCalled();
    await resolveShowPrompt();
    stdin.emit(Number('z'.charCodeAt(0)).toString(16));
    expect(showPrompt2).toHaveBeenCalled();
  });

  it('Pressing "o" runs test in "only changed files" mode', () => {
    watch(globalConfig, contexts, pipe, hasteMapInstances, stdin);
    runJestMock.mockReset();

    stdin.emit(KEYS.O);

    expect(runJestMock).toBeCalled();
    expect(runJestMock.mock.calls[0][0].globalConfig).toMatchObject({
      onlyChanged: true,
      watch: true,
      watchAll: false,
    });
  });

  it('Pressing "a" runs test in "watch all" mode', () => {
    watch(globalConfig, contexts, pipe, hasteMapInstances, stdin);
    runJestMock.mockReset();

    stdin.emit(KEYS.A);

    expect(runJestMock).toBeCalled();
    expect(runJestMock.mock.calls[0][0].globalConfig).toMatchObject({
      onlyChanged: false,
      watch: false,
      watchAll: true,
    });
  });

  it('Pressing "ENTER" reruns the tests', () => {
    watch(globalConfig, contexts, pipe, hasteMapInstances, stdin);
    expect(runJestMock).toHaveBeenCalledTimes(1);
    stdin.emit(KEYS.ENTER);
    expect(runJestMock).toHaveBeenCalledTimes(2);
  });

  it('Pressing "t" reruns the tests in "test name pattern" mode', async () => {
    const hooks = new JestHooks();

    watch(globalConfig, contexts, pipe, hasteMapInstances, stdin, hooks);
    runJestMock.mockReset();

    stdin.emit(KEYS.T);
    ['t', 'e', 's', 't'].map(toHex).forEach(key => stdin.emit(key));
    stdin.emit(KEYS.ENTER);
    await nextTick();

    expect(runJestMock.mock.calls[0][0].globalConfig).toMatchObject({
      testNamePattern: 'test',
      testPathPattern: '',
      watch: true,
      watchAll: false,
    });
  });

  it('Pressing "p" reruns the tests in "filename pattern" mode', async () => {
    const hooks = new JestHooks();

    watch(globalConfig, contexts, pipe, hasteMapInstances, stdin, hooks);
    runJestMock.mockReset();

    stdin.emit(KEYS.P);
    ['f', 'i', 'l', 'e'].map(toHex).forEach(key => stdin.emit(key));
    stdin.emit(KEYS.ENTER);
    await nextTick();

    expect(runJestMock.mock.calls[0][0].globalConfig).toMatchObject({
      testNamePattern: '',
      testPathPattern: 'file',
      watch: true,
      watchAll: false,
    });
  });

  it('Can combine "p" and "t" filters', async () => {
    const hooks = new JestHooks();

    watch(globalConfig, contexts, pipe, hasteMapInstances, stdin, hooks);
    runJestMock.mockReset();

    stdin.emit(KEYS.P);
    ['f', 'i', 'l', 'e'].map(toHex).forEach(key => stdin.emit(key));
    stdin.emit(KEYS.ENTER);
    await nextTick();

    stdin.emit(KEYS.T);
    ['t', 'e', 's', 't'].map(toHex).forEach(key => stdin.emit(key));
    stdin.emit(KEYS.ENTER);
    await nextTick();

    expect(runJestMock.mock.calls[1][0].globalConfig).toMatchObject({
      testNamePattern: 'test',
      testPathPattern: 'file',
      watch: true,
      watchAll: false,
    });
  });

  it('Pressing "u" reruns the tests in "update snapshot" mode', async () => {
    const hooks = new JestHooks();

    globalConfig.updateSnapshot = 'new';

    watch(globalConfig, contexts, pipe, hasteMapInstances, stdin, hooks);
    runJestMock.mockReset();

    hooks.getEmitter().testRunComplete({snapshot: {failure: true}});

    stdin.emit(KEYS.U);
    await nextTick();

    expect(runJestMock.mock.calls[0][0].globalConfig).toMatchObject({
      updateSnapshot: 'all',
      watch: true,
      watchAll: false,
    });

    stdin.emit(KEYS.A);

    await nextTick();
    // updateSnapshot is not sticky after a run.
    expect(runJestMock.mock.calls[1][0].globalConfig).toMatchObject({
      updateSnapshot: 'new',
      watch: false,
      watchAll: true,
    });

    results = {snapshot: {failure: true}};

    stdin.emit(KEYS.A);
    await nextTick();

    runJestMock.mockReset();
    stdin.emit(KEYS.U);
    await nextTick();

    expect(runJestMock.mock.calls[0][0].globalConfig).toMatchObject({
      updateSnapshot: 'all',
      watch: false,
      watchAll: true,
    });
  });

  it('passWithNoTest should be set to true in watch mode', () => {
    globalConfig.passWithNoTests = false;
    watch(globalConfig, contexts, pipe, hasteMapInstances, stdin);
    globalConfig.passWithNoTests = true;
    expect(runJestMock.mock.calls[0][0]).toMatchObject({
      globalConfig,
    });
  });
});

class MockStdin {
  constructor() {
    this._callbacks = [];
  }

  setRawMode() {}

  resume() {}

  setEncoding() {}

  on(evt, callback) {
    this._callbacks.push(callback);
  }

  emit(key) {
    this._callbacks.forEach(cb => cb(key));
  }
}
