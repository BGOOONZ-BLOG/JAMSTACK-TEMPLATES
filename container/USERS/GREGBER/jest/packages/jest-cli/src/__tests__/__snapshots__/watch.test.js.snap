// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Watch mode flows Runs Jest in a non-interactive environment not showing usage 1`] = `
Array [
  "
",
]
`;

exports[`Watch mode flows Runs Jest once by default and shows usage 1`] = `
Array [
  "
Watch Usage
 › Press a to run all tests.
 › Press f to run only failed tests.
 › Press p to filter by a filename regex pattern.
 › Press t to filter by a test name regex pattern.
 › Press q to quit watch mode.
 › Press Enter to trigger a test run.
",
]
`;

exports[`Watch mode flows shows prompts for WatchPlugins in alphabetical order 1`] = `
Array [
  Array [
    "
Watch Usage
 › Press a to run all tests.
 › Press f to run only failed tests.
 › Press p to filter by a filename regex pattern.
 › Press t to filter by a test name regex pattern.
 › Press q to quit watch mode.
 › Press s to do nothing.
 › Press u to do something else.
 › Press Enter to trigger a test run.
",
  ],
]
`;

exports[`Watch mode flows shows update snapshot prompt (with interactive) 1`] = `
Array [
  Array [
    "
Watch Usage
 › Press a to run all tests.
 › Press f to run only failed tests.
 › Press p to filter by a filename regex pattern.
 › Press t to filter by a test name regex pattern.
 › Press u to update failing snapshots.
 › Press i to update failing snapshots interactively.
 › Press q to quit watch mode.
 › Press Enter to trigger a test run.
",
  ],
]
`;

exports[`Watch mode flows shows update snapshot prompt (without interactive) 1`] = `
Array [
  Array [
    "
Watch Usage
 › Press a to run all tests.
 › Press f to run only failed tests.
 › Press p to filter by a filename regex pattern.
 › Press t to filter by a test name regex pattern.
 › Press u to update failing snapshots.
 › Press q to quit watch mode.
 › Press Enter to trigger a test run.
",
  ],
]
`;
